### PR TITLE
readme: clarify conf syntax, format deps into list, decrease logo size

### PR DESCRIPTION
**Pull request by radfish**
#951

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,7 +42,12 @@ The Bitcoin donation address is: 1KTexdemPdxSBcG55heUuTjDRYqbC5ZL8H
...
+[<img width="80" src="https://static.getmonero.org/images/sponsors/mymonero.png"/>](https://mymonero.com)
+[<img width="150" src="https://static.getmonero.org/images/sponsors/kitware.png?1"/>](http://kitware.com)
+[<img width="100" src="https://static.getmonero.org/images/sponsors/dome9.png"/>](http://dome9.com)
+[<img width="150" src="https://static.getmonero.org/images/sponsors/araxis.png"/>](http://araxis.com)
+[<img width="150" src="https://static.getmonero.org/images/sponsors/jetbrains.png"/>](http://www.jetbrains.com/)
+[<img width="150" src="https://static.getmonero.org/images/sponsors/navicat.png"/>](http://www.navicat.com/)
```
Comment (moneromooo-monero) [inline README.md]: In the README ? Does something parse this to display those images from the README ?

File: README.md [hunk 2]
```diff
@@ -54,8 +59,21 @@ See [LICENSE](LICENSE).
...
-Dependencies: GCC 4.7.3 or later, CMake 3.0.0 or later, libunbound 1.4.16 or later (note: Unbound is not a dependency, libunbound is), libevent 2.0 or later, libgtest 1.5 or later, and Boost 1.53 or later (except 1.54, [more details here](http://goo.gl/RrCFmA)), BerkeleyDB 4.8 or later (note: on Ubuntu this means installing libdb-dev and libdb++-dev).
-Static Build Additional Dependencies: ldns 1.6.17 or later, expat 1.1 or later, bison or yacc
+Dependencies:
+
+* GCC `>=4.7.3`
+* CMake `>=3.0.0`
+* libunbound `>=1.4.16` (note: Unbound is not a dependency, libunbound is)
+* libevent `>=2.0`
+* libgtest `>=1.5`
+* Boost `>=1.53 && !=1.54` (note: 1.54 is not supported [more details here](http://goo.gl/RrCFmA)),
```
Comment (moneromooo-monero) [inline README.md]: Might as well say >= 1.58, there's another PR that will bump this very soon

**Comment by grummerd** [comment]
please [add dependency pkg-config](https://github.com/moneroexamples/compile-monero-09-on-ubuntu-16-04/issues/1#event-762046472)

[See current dependencies list](https://github.com/moneroexamples/compile-monero-09-on-ubuntu-16-04)

File: README.md (hunk 1 above)
Comment (radfish) [inline README.md]: Yeah, it's markdown. See here for what it looks like and compare with master:
https://github.com/radfish/bitmonero/tree/PR--readme-clarify-conf-syntax

**Comment by radfish** [comment]
@grummerd Added some more dependencies

**Comment by grummerd** [comment]
Another super minor doc suggestion

At the bottom of the docs in section, `using readline`. Any complete commands should be on separate line, not part of the text. Added benefit helps to break up a wall of text. btw i didn't format the everything nicely in markdown. Just blockquote the last line. Sorry

Using readline

While bitmonerod and simplewallet do not use readline directly, most of the functionality can be obtained by running them via rlwrap. This allows command recall, edit capabilities, etc. It does not give autocompletion without an extra completion file, however. To use rlwrap, simply prepend "rlwrap " to the command line, eg: rlwrap bin/simplewallet --wallet-file /path/to/wallet

becomes...

Using readline

While bitmonerod and simplewallet do not use readline directly, most of the functionality can be obtained by running them via rlwrap. This allows command recall, edit capabilities, etc. It does not give autocompletion without an extra completion file, however. To use rlwrap, simply prepend "rlwrap" to the command line, 

> rlwrap bin/simplewallet --wallet-file /path/to/wallet

**Comment by grummerd** [comment]
> clarify conf syntax


Suggest including a default conf file. Assume

- port 18080 is opened and p2p host address 127.0.0.1

or 

- If want UPnP as the default,  include a commented out line for the p2p port #

Have the conf file install to the default location (correct me if i'm wrong, doing this from memory)


~/.bitmonero/bitmonero.conf

**Comment by radfish** [comment]
Applied your formatting suggestions, thanks.
Example config file is too big a topic for this PR, won't deal with it here.

**Comment by fluffypony** [comment]
Reviewed

**Pull request merged**